pkg/client/client.go

```diff
@@ -564,10 +645,33 @@ func (c *KpmClient) VendorDeps(kclPkg *pkg.KclPkg) error {
 
 // FillDepInfo will fill registry information for a dependency.
 func (c *KpmClient) FillDepInfo(dep *pkg.Dependency) error {
-	if dep.Source.Oci != nil {
+	if dep.Source.Git == nil {
```

Comment:
Why not deal git deps here?

Reply:
At present, the repository from git can not pull metadata, because the metadata is dependent on a specific repo, not from git. We can consider adding the integration of gitlab or github, which can be put in the future work.